skyportal/handlers/api/group.py

```diff
@@ -136,22 +143,34 @@ def get(self, group_id=None):
             ) and group.id not in [g.id for g in self.current_user.accessible_groups]:
                 return self.error('Insufficient permissions.')
 
+            if self.get_query_argument("includeGroupUsers", "true").lower() in (
```

Comment:
when you pass in `includeGroupUsers=false`, does that get converted to the Python boolean `False`, or just the string `"false"`?

Reply:
nm, just checked, this looks good